doc/rst/technotes/initializers.rst

```diff
@@ -12,123 +12,289 @@ than the original methods known as constructors.
 A discussion of the current design and rationale is provided in
 `CHIP 10 <https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/chips/10.rst>`_.
 
-Release Chapel 1.15.0 provides a strong preview implementation of this
-new feature. This initial implementation is not of production quality
-and has not been heavily tested, but the basic features are working
-and it is believed that adventurous developers will have success using
-init methods in new code.
+Release Chapel 1.16.0 provides a strong preview implementation of this
+new feature.  Though some known bugs remain, the feature is rapidly
+approaching full support and developers should feel encouraged to
+write initializers where previously they had relied on constructors.
 
 It is anticipated that the implementation will continue to advance
 steadily after this release and that many aspects of the internal
 implementation of Chapel will transition from constructor methods to
 initializer methods during this release.
 
 
-Stability
----------
+The Initializer Method
+----------------------
 
-The functionality of an initializer method has four major aspects
+An initializer is a method on a class or record named "init".  It is invoked
+by the ``new`` operator, where the type name and initializer arguments are
+preceded with the ``new`` keyword.
 
-  - user-defined code that runs in phase 2
+If the program declares a type initializer method, it is a user-defined
+initializer.  If the program declares no initializers or constructors for a
+class or record, a compiler generated constructor for that type is created
+automatically (see below for the status on `Compiler Generated Initializers`_).
 
-  - user-defined code that runs in phase 1
+User-Defined Initializers
+-------------------------
 
-  - compiler generated code that runs in phase 1
+A user-defined initializer is an initializer method explicitly declared in the
+program.  An initializer declaration has the same syntax as a method
+declaration, except that the name of the function is "init", and there is no
+return type specifier.
 
-  - the generation of effective user-facing diagnostic messages
+When an initializer is called, the usual function resolution mechanism is
```

Comment:
I am unsure if "When an initializer is called," in this context might be confusing or not.  Some readers
may have skimmed the previous text that speaks of new expressions and uses the word "invoked"
and may fail to be clear that typical user code should not call init() directly.

Reply:
That's fair.  I stole those lines directly from the constructor section of the spec, should I update it there as well?